app/src/main/java/org/fossasia/phimpme/gallery/util/ThemeHelper.java

```diff
@@ -125,6 +125,22 @@ public int getBackgroundColor() {
     return color;
   }
 
+  public int getUpdatedSwitchColor() {
+    int color;
+    switch (baseTheme) {
+      case DARK_THEME:
+        color = getColor(R.color.md_dark_background);
+        break;
+      case AMOLED_THEME:
+        color = getColor(R.color.md_brown_300);
```

Comment:
Can you check the material color guide and maybe get a more interactive and interesting color?

Reply:
sure @abishekvashok  i will look for better colour in material colour guide

Reply:
Cool then :D

Reply:
@abishekvashok  please review changes